Scripts/Python/kdshTreeRingsSolution.py

```diff
@@ -300,9 +299,9 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
             #~ PtDebugPrint("TRS: Nothing to ff. VARname = ", VARname)
             pass
         else:
-            GUIcode = "GUI" + string.join(string.split(VARname, "Ring"), "") + "_0" + str(newbearing) + ".animation.play()"
-            #~ PtDebugPrint("GUIcode = ", GUIcode)
-            exec(GUIcode) # this runs the animation on the "fake" ring in front of the GUI
+            # this runs the animation on the "fake" ring in front of the GUI
+            animname= "GUI_{id}_0_{bearing}".format(id="".join(VARname.split("Ring")), bearing=newbearing)
```

Comment:
There seem to be a couple of extra `_` here...  Shouldn't it be `GUI{id}_0{bearing}`?

Reply:
indeed!